xml/s4s_tune.xml

```diff
@@ -425,7 +492,7 @@
   <note>
    <title>No Profiles in &productname; &productnumber;</title>
    <para>
-    In &productname; 15 and above &sapconf; does not support profiles anymore.
+    In &productname; 15, &sapconf; does not support profiles anymore.
```

Comment:
```suggestion
    In &productname; 15, &sapconf; no longer supports profiles.
```

Reply:
By the way: "any more" is the phrase you wanted.

Reply:
Thanks Liam, applied. :+1: